lib/core/msdf-sys/src/test_utils.rs

```diff
@@ -0,0 +1,31 @@
+use std::pin::Pin;
+use std::task::{Context, Poll};
+use std::future::Future;
+use crate::{ is_emscripten_runtime_initialized, run_once_initialized };
+
+/// The future for running test after initialization
+pub struct TestAfterInit<F:Fn()> {
+    test : F
+}
+
+impl<F: Fn()> TestAfterInit<F> {
+    pub fn schedule(test:F) -> TestAfterInit<F> {
+        TestAfterInit{test}
+    }
+}
+
+impl<F : Fn()> Future for TestAfterInit<F> {
```

Comment:
Btw I'm leaving this comment just to make sure we are on the same page regarding general style guide for future. I mean, I'm completely ok with accepting code which occasionally doesnt follow it, because requiring good formatting always from human being (not a machine) is stupid, but I just want to confirm - according to guide, this should be written as follow, right?
```rust
impl<F:Fn()> Future for TestAfterInit<F> {
```

Reply:
Yes, this is a line I wrote before coding guide, and overlooked then